ui/app/src/components/ContentTypeManagement/components/TypeCard.tsx

```
@@ -0,0 +1,95 @@
+					},
+					subheader: { variant: 'body2', noWrap: 'true', title: skeleton ? undefined : type.id },
+					content: { sx: { overflow: 'hidden' } }
+				}}
+			/>
+			<TypeCardMedia skeleton={skeleton} typeId={type?.id} sx={styleOverrides?.cardMedia} />
+		</>
+	);
+	return (
+		<Card {...cardProps} sx={consolidateSx(baseCardSx, !hasActionArea && styleOverrides?.actionArea, sx)}>
+			{hasActionArea ? (
+				<CardActionArea sx={styleOverrides?.actionArea} onClick={(e) => onClick(e, type)} children={cardBody} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)